doc/modules/ensemble.rst

```diff
@@ -1033,3 +1032,90 @@ The following example shows how to fit the VotingRegressor::
 .. topic:: Examples:
 
   * :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`
+
+.. _stacking:
+
+Stacked generalization
+----------------------
+
+Stacked generalization is a method for combining estimators to reduce their
+biases [W1992]. More precisely, the predictions of each individual estimator
+are stacked together and used by another estimator to compute the final
+prediction. This final estimator is trained through cross-validation.
+
+The :class:`StackingClassifier` and :class:`StackingRegressor` provide such
+strategies which can be applied to classification and regression problems.
+
+The ``estimators`` parameter corresponds to the list of the estimators which
+are stacked together. It should be given as a list of name and estimator::
+
+  >>> from sklearn.linear_model import RidgeCV, LassoCV
+  >>> from sklearn.svm import SVR
+  >>> estimators = [('ridge', RidgeCV()),
+  ...               ('lasso', LassoCV(random_state=42)),
+  ...               ('svr', SVR(C=1, gamma=1e-6, kernel='rbf'))]
+
+The ``final_estimator`` will combine the predictions of the ``estimators``. It
+needs to be a classifier or a regressor when using :class:`StackingClassifier`
+or :class:`StackingRegressor`, respectively::
+
+  >>> from sklearn.ensemble import GradientBoostingRegressor
+  >>> from sklearn.ensemble import StackingRegressor
+  >>> reg = StackingRegressor(
+  ...     estimators=estimators,
+  ...     final_estimator=GradientBoostingRegressor(random_state=42))
+
+To train the ``estimators`` and ``final_estimator``, the ``fit`` method needs
+to be called on the training data::
+
+  >>> from sklearn.datasets import load_boston
+  >>> X, y = load_boston(return_X_y=True)
+  >>> from sklearn.model_selection import train_test_split
+  >>> X_train, X_test, y_train, y_test = train_test_split(X, y,
+  ...                                                     random_state=42)
+  >>> reg.fit(X_train, y_train)  # doctest: +ELLIPSIS
+  StackingRegressor(...)
+
+During training, the ``estimators`` are fitted on the whole training data
+``X_train``. They will be used when calling ``predict``, ``predict``, or
+``predict_proba``. To generalize and avoid over-fitting, the
+``final_estimator`` is trained on out-samples using internally
+:func:`sklearn.model_selection.cross_val_predict`.
+
+Note that the output of the ``estimators`` is controlled by the parameter
+``method_estimators``. It corresponds to the method called by each
+estimator. This parameter is either a list of string of the methods name or
+``'auto'`` which will automatically called the method depending of the
+availability and a pre-determined order of preference.
+
+It is possible to get the final predictions using the ``predict`` or
+``predict_proba`` depending if you are using :class:`StackingClassifier` or
+:class:`StackingRegressor`::
+
+   >>> y_pred = reg.predict(X_test)
```

Comment:
That example does not really illustrate the text above. May I suggest

> The fitted stacking estimators have a `predict` method, and classifier also have a `predict_proba` method. They can be used as any other estimator, e.g.::`

Or something like that (also, use single ticks)